tests/test_post_processings_results.py

```diff
@@ -62,14 +62,18 @@ def test_post_processing_result_global_coefficients(simai_client):
                             "X": 4.441544089820862,
                             "Y": 0.32894839148911376,
                             "Z": 14.442021446988662,
-                        }
+                        },
+                        "type": "base",
+                        "unit": {"N": 1},
```

Comment:
Not a big deal but is "unit" useful here? As far I know it is never used.

Reply:
I put it in the test because that's the current content of the global coefficients. Whether it should be there or not is not a question for me.